### ISSUE TITLE
Prep JMS binder for 1.1 release

### ISSUE THREAD

**Issue by sabbyanandan** [spring-cloud/spring-cloud-stream-binder-jms#6]
- add docs infrastructure
- use the generic test infrastructure for incremental tests

**Comment by cforce** [comment]
There is less to zero activity since march. When will the project be released?

**Comment by chsi13** [comment]
Any news ??? Really interested about this.

**Comment by olegz** [comment]
We are considering it but at the moment there are no plans yet, but we'll revisit it shortly

**Comment by chsi13** [comment]
Current version is in Alpha.  Maybe I can fork it , try it and contribute on it. Prefer avoid to do it directly with Spring integration JMS.

**Comment by olegz** [comment]
Pull requests are definitely welcomed. Why do you say you prefer to stay away from Spring Integration JMS?

**Comment by chsi13** [comment]
We want to implement a choreography-based event model. But we don't have the possibility to use a broker like rabbitMQ directly (it's enterprise politic ;)). And we found that this project really simplify the writing of these flow. Microservices consumes an event, process it and produce another event for another consumer service. That's our model and spring cloud stream fits great for this. What do you think ?

**Comment by olegz** [comment]
I am not sure that answers my question. Spring Integration JMS provides most of the code that you would need within the binder. We use Spring Integration AMQP and Spring Integration Kafka in both Rabbit and Kafka binders.  Basically code reusability where the use of respective Inbound and Outbound adapters allows for a quick binding between the source destination and the channels identified and bound by `Source`, `Processor` and/or `Sink`.
Did you have something else in mind?

**Comment by chsi13** [comment]
No I know that spring cloud stream is an abstraction on Spring Integration JMS. I just found really great the simplified model that SCS brings on the development of event-driven microservices architecture. So sorry for badly expressed statement. I just want to say that I prefer develop my microservices with the abstraction of SCS instead do it directly with spring jms. Clear for you ?

**Comment by olegz** [comment]
Yep, got it. So would be interested in reviewing your contribution.

**Comment by chsi13** [comment]
Yes do it soon ;-). Thx for answers.
